articles/mdm-migration.md

```diff
@@ -2,46 +2,174 @@
+
+3. Under **Mode** choose a mode and enter the webhook URL for your automation tool (ex. Tines) under **Webhook URL** and select **Save**.
+
+4. During the end user migration workflow, an end user's device will have their selected system theme (light or dark) applied. If your logo is not easy to see on both light and dark backgrounds, you can optionally set a logo for each theme:
+Head to **Settings** > **Organization settings** >
+**Organization info**, add URLs to your logos in the **Organization avatar URL (for dark backgrounds)** and **Organization avatar URL (for light backgrounds)** fields, and select **Save**.
+
+fleetctl CLI:
+
+1. Create `fleet-config.yaml` file or add to your existing `config` YAML file:
+
+```yaml
+apiVersion: v1
+kind: config
+spec:
+  mdm:
+    macos_migration:
+      enable: true
+      mode: "voluntary"
+      webhook_url: "https://example.com"
+  ...
+```
+
+2. Fill in the above keys under the `mdm.macos_migration` key.
+
+To learn about each option, in the Fleet UI, select the avatar on the right side of the top navigation, select **Settings > Integrations > Mobile device management (MDM)**, and scroll down to the **End user migration workflow** section.
+
+3. During the end user migration workflow, the window will show the Fleet logo on top of a dark and light background (appearance configured by end user).
+
+If you would like to add your organization's logo, you can optionally set a logo for each background:
+
+```yaml
+apiVersion: v1
+kind: config
+spec:
+  org_info:
+    org_logo_url: https://fleetdm.com/images/press-kit/fleet-blue-logo.png
+    org_logo_url_light_background: https://fleetdm.com/images/press-kit/fleet-white-logo.png
+  ...
+```
+
+Add URLs to your logos that are visible on a dark background and light background in the `org_logo_url` and `org_logo_url_light_background` keys respectively. If you only set a logo for one, the Fleet logo will be used for the other.
+
+1. Run the fleetctl `apply -f fleet-config.yml` command to add your configuration.
+
+2. Confirm that your configuration was saved by running `fleetctl get config`.
+
+3. Send [these guided instructions](#how-to-turn-on-mdm-end-user-manual-migration) to your end users to complete the final few steps via Fleet Desktop.
+
+##### How to turn on MDM (end user)
+
+1. Select the Fleet icon in your menu bar and select **Migrate to Fleet**.
+
+2. Select **Start** in the **Migrate to Fleet** popup.
+
+2. On your **My device** page, select the **Turn on MDM** button in the yellow banner and follow the instructions.
+    * If you don’t see the yellow banner or the **Turn on MDM** button, select the purple **Refetch** button at the top of the page.
+    * If you still don't see the **Turn on MDM** button or the **My device** page presents you with an error, please contact your IT administrator.
+
+
+## Migrate automatically enrolled (ADE) hosts
```

Comment:
```suggestion
## Migrate automatically enrolled (DEP) hosts
```

I think we use "DEP" in other spots in the product and docs. I could be wrong but worth double checking. IT admins are still used to DEP and we use it in internal/external convos.

Reply:
Ah gotcha! I assumed we were on "ADE" now, but lemme double check other docs we have.

Reply:
Yeah, looks like we use "DEP" a lot more than we use "ADE" in our docs. I will switch back to "DEP" for clarity

Reply:
It's been almost a decade since Apple renamed this. We can use ADE.

Reply:
Happy to go either way! just lmk

Reply:
Let's match Apple's language and use ADE.

Reply:
Hey! I think let's prioritize consistency here and go w/ "DEP" for now.

At some point, we decided to use "DEP" b/c that's what we used in internal/external conversations. I think "DEP" is all over the product and docs.

It might be time to change to "ADE" but I think let's address that language change as a whole (everywhere) in a separate pass.

@jahzielv can you please help track a feature request for that so we don't lose it?

Reply:
Excluding release notes, we use DEP in 3 other articles and ADE in 5 articles. Let's match Apple's language.

Reply:
ADE sounds good to me 👍 

@spokanemac can you please help track a feature request to change it elsewhere? I think the important thing is we don't forget to make it consistent.